### ISSUE TITLE
Playing a Secret with modified Mana Cost shows the original Mana Cost for your opponent

### ISSUE THREAD

**Issue by Patashu** [HearthSim/hs-bugs#93]
Playing a Secret that has its mana cost modified will display its original cost (only for the opponent) on the card when it enters the play. This is probably because playing Secrets will always show the picture that says "This is hidden until a secret action occurs... Watch out!"
http://imgur.com/hZJhswR (Ice Barrier for 2 mana)
http://imgur.com/owxOUC7 (Mirror Entity for 8 mana)

**Comment by ZeroRin** [comment]
tested in 3.1, fixed.
though the color of the cost is still white
http://tieba.baidu.com/p/4084328669

**Comment by culinko** [comment]
Repurposed the ticket. Previously, the original mana cost was displayed, they fixed it in 3.1, but didn't change the mana cost color to green/red. So now the issue is only about the incorrect white color on the mana cost.

**Comment by culinko** [comment]
This issue can't be verified anymore until #668 is fixed.

**Comment by troggnostupidhs** [comment]
I believe #668 is fixed

**Comment by culinko** [comment]
Yeah, #668 is fixed now and I just verified that this issue is still present.

**Comment by troggnostupidhs** [comment]
Deck
AAEBAf0EAtYRp80CDnGzAcMB+wGcArQCqwTmBJYFuQaYDbdswcEC4soCAA==
